README.md

```diff
@@ -65,6 +65,13 @@ var sortable = new Sortable(el, {
 	ghostClass: "sortable-ghost",  // Class name for the drop placeholder
 	dataIdAttr: 'data-id',
 	
+	/*
+	 ignore the HTML5 DnD behaviour and force the fallback to kick in
+	 - provide a more reliable cross-browser solution
+	 - grants the ability to display a different "dragged" element
+	*/
+	forcePolyfill: false,
```

Comment:
Yet the `forceFallback` is more correctly, because the library does not contain `polyfill`.